src/posts/2021-02-15-phaser-blog.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Phaser'
+comments: true
+author: eline
+date: 2020-02-15
+---
```

Comment:
Hier mist nog een `min_read`. Weet niet of dit een vereiste is maar de andere posts hebben dit wel